Sources/Kevin/Accounts/AccountLinking/KevinAccountLinkingViewController.swift

```diff
@@ -31,6 +32,10 @@ internal class KevinAccountLinkingViewController :
         )
     }
     
+    override func onCloseTapped() {
+        dismiss(animated: true, completion: onExit)
```

Comment:
💡 It seems `Close` button tap means cancellation of Account Linking flow. If so, then maybe `onCancel` would be a better name for callback? 🤓

Reply:
This function is related to UIViewController lifecycle behaviour, and does not know much about AccountLinking flow. This is mostly handled by `KevinAccountLinkingSession` which decides what is happening with flow. 
I have ,however, changed closure name to `onClose` for parity.